WebApp/package-lock.json

```diff
@@ -2449,7 +2449,7 @@
         },
         "opn": {
           "version": "5.3.0",
-          "resolved": "https://registry.npmjs.org/opn/-/opn-5.3.0.tgz",
+          "resolved": "http://registry.npmjs.org/opn/-/opn-5.3.0.tgz",
```

Comment:
This file is part of your fixcture?

Reply:
No, but without update this file frontend doesn't work well

Reply:
So it fixes somethink else and it should be in some other commit and pull request